driver/src/main/scala/api/AsyncDriver.scala

```diff
@@ -18,11 +18,11 @@ import reactivemongo.core.actors.{
   StandardDBSystemWithX509
 }
 import reactivemongo.core.nodeset.Authenticate
-
-import akka.actor.{ Actor, ActorRef, ActorSystem, Props, Terminated }
-import akka.pattern.ask
-import akka.util.Timeout
+import reactivemongo.actors.actor.{ Actor, ActorRef, ActorSystem }
```

Comment:
import order (`scalafix`)

Reply:
Addressed in 87d3301898f6bbf86f7167a2c141cc2bbd9e8370